### DISCUSSION TITLE
error during copy-dbalogins

### DISCUSSION THREAD

**Discussion by theviking28** [dataplat/dbatools#7629] [Q&A] [answered]
I installed dbatools using powershell on my windows 10 pc.  I opened powershell with an account which is a windows login and has sysadmin rights on 2 sql servers.  

Then I attempted to use the command “copy-dbalogin -source sql1 -destination sql2     -- as far as I can tell this is correct to migrate logins over.  

I get the error “error occurred while establishing connection to sql2 | the system can not find the file specified”

Source sql is 2014, destination is 2019. 

What should I be trying to look at to resolve?  Only thing I can think of is that on the new 2019 sql, I installed sql itself to the E: drive and not default C:   Does dbatools rely on or expect a default C: installation?  If so is there any command line switches to use to specify differently?

**Comment by potatoqualitee** [comment, marked as answer]
hey @theviking28 - that is a named pipes connection issue (it cant find your named pipe). not sure why it's attempting named pipes though you do have a weird character at the end. you can force tcp with `sql2:1433` and that should take care of it.
